opendm/config.py

```diff
@@ -17,28 +31,34 @@ def __call__(self, parser, namespace, values, option_string=None):
 
 def config():
     parser.add_argument('--images', '-i',
-                        metavar='<string>',
+                        metavar='<path>',
                         help='Path to input images'),
 
     parser.add_argument('--project-path',
-                        metavar='<string>',
-                        help='Path to the project to process')
+                        metavar='<path>',
+                        help='Path to the project folder',
+                        default=defaultSettings['project_path'])
+
+    parser.add_argument('name',
+                        metavar='<project name>',
+                        type=alphanumeric_string,
+                        help='Name of Project (i.e subdirectory of projects folder)')
 
     parser.add_argument('--resize-to',  # currently doesn't support 'orig'
                         metavar='<integer>',
-                        default=2400,
+                        default=defaultSettings['resize_to'],
```

Comment:
I was trying to avoid using a dictionary for args, but it seems like more trouble to keep it as a Namespace. If I do that then you are correct, I should be putting it in config()